static/src/components/ImageDisplay.vue

```diff
@@ -16,16 +18,25 @@ export default defineComponent({
     imageId: { type: String, required: true },
   },
 
-  computed: {
-    src() {
-      return getUrl(`tasks/${this.taskId}/${this.imageId}`);
-    },
+  data() {
+    return {
+      src: [ref<string | undefined>(undefined)],
+    };
+  },
+
+  mounted() {
+    this.src = [
+      computedAsync(async () => {
+        let image = await API.getImage(this.taskId, this.imageId);
+        return image ? URL.createObjectURL(image) : undefined;
+      }, undefined),
+    ];
```

Comment:
Sorry I forgot, but this returns the image so we dont have to use the url in the `<img>` tag which does not work due to authorization right? Nice that you were able to integrate that.
A possible solution for the async thing might be vue asyncComponents. But im fine if we put that in a different pr

Reply:
Let's move it to a separate issue, there's already one: #62